dspace-api/src/main/java/org/dspace/app/bulkedit/MetadataExportSearchCli.java

```diff
@@ -0,0 +1,12 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+
+package org.dspace.app.bulkedit;
+
+public class MetadataExportSearchCli extends MetadataExportSearch {
```

Comment:
Please add JavaDocs to describe.  It's required for all classes, even if the class is empty.